### ISSUE TITLE
[shell] shell-history does not exist for Docker

### ISSUE THREAD

**Issue by joverlee521** [nextstrain/cli#231]
### Current Behavior
When running `nextstrain shell` with Docker, the command errors if the shell-history file does not exist:
```
$ nextstrain shell .
Entering the Nextstrain build environment

Mapped volumes:
  /nextstrain/build is from /Users/jlee2346/Repos/nextstrain

Run the command "exit" to leave the build environment.

Error: The path(s) given for the following components do not exist
or are not directories:

    • bash_history: /Users/jlee2346/.nextstrain/shell-history
```

### Expected behavior
The shell-history file should be created prior to mounting in Docker. 
Should the cli automatically create the file behind the scenes or should we just leave it to users to create it?

**Comment by tsibley** [comment]
Oh, bummer. This is a regression in 5.0.0 compared to 4.2.0. The error comes from `nextstrain.cli.runner.docker.run()`, which checks that all mounted volumes from the host already exist:

https://github.com/nextstrain/cli/blob/4e1960bb64b039d54fc9a4d52c4504c7004613cd/nextstrain/cli/runner/docker.py#L63-L79

**Comment by tsibley** [comment]
I think `nextstrain.cli.command.shell.run()` should explicitly ensure that the `history_file` it arranges exists with something like:

```python
history_file = SHELL_HISTORY.resolve(strict = True)
history_file.parent.mkdir(parents = True, exist_ok = True)

try:
    history_file.touch()
except FileExistsError:
    pass
```

**Comment by tsibley** [comment]
Should be fixed by #232.

**Comment by tsibley** [comment]
Fix will be released as 5.0.1 when it gets thru CI.
